src/pocketdb/repositories/ChainRepository.h

```diff
@@ -45,6 +45,10 @@ namespace PocketDb
         // Erase all calculated data great or equals block
         bool Rollback(int height);
 
+        // Get hash value of Transaction of a given row in the database, used to validate
```

Comment:
A bit frustrating description, it gets hashes of all transactions, not of a given row.

Reply:
This is only called from my block test for right now, so I will remove it from here and move it into pocketnet_block_tests.cpp.